docs/resources/permissionassignment.md

```diff
@@ -0,0 +1,49 @@
+---
+subcategory: "Unity Catalog"
+---
+# databricks_permissionassignment Resource
+
+TBA
+
+These resources are invoked in the workspace context.
+
+## Example Usage
+
+Adding workspace group
+
+```hcl
+resource "databricks_group" "data_eng" {
+  display_name = "Data Engineering"
+}
+
+resource "databricks_permissionassignment" "add_admin_group" {
+  principal_id = databricks_group.data_eng.id
+  permissions  = ["ADMIN"]
+}
+```
+
+Adding workspace user
```

Comment:
If I understand correctly, this is trying to grant permission to 
subject : account level user 
resource: workspace 
permission level : USER

Why are we calling this as adding workspace user?

Reply:
i think it's because of unclarity of workspace-level API in internal docs... let's see after integration testing today